### PR TITLE
Bug 1748777: bump kubernetes-kubectl

### PR DESCRIPTION
**Pull request by sallyom**
/assign @soltysh 

bump k8s.io/kubectl, then `go mod vendor` to bring in this:
https://github.com/openshift/kubernetes-kubectl/pull/3
and https://github.com/openshift/kubernetes-kubectl/pull/5

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -115,7 +115,7 @@ replace (
 	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.0.0-20190918162944-7a93a0ddadd8
 	k8s.io/kube-proxy => k8s.io/kube-proxy v0.0.0-20190918162534-de037b596c1e
 	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.0.0-20190918162820-3b5c1246eb18
-	k8s.io/kubectl => github.com/openshift/kubernetes-kubectl v0.0.0-20190926190201-54079662af88
+	k8s.io/kubectl => github.com/sallyom/kubernetes-kubectl v0.0.0-20191031164730-0a8b1aff5bc5
```
Comment (sallyom) [inline go.mod]: for testing, will change one that branch merges

**Comment by sallyom** [comment]
@soltysh my `go mod vendor` removed this file, https://github.com/openshift/oc/blame/982438ffba465bc85b40320e307a1d4598348692/vendor/k8s.io/kubectl/pkg/cmd/create/patch_scc.go   but we need it for e2es, how do i add it back? : )

**Comment by soltysh** [comment]
Superseded by #144 
@sallyom point all your BZs that need it to the other PR.
/close

**Pull request closed without merging**